react-time-track/src/utils/calculateProgress.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Calculates a project's progress percentage considering only the consumed budget (project.real_cost)
+ * @param {String} real - should get project.real_cost
+ * @param {String} estimated - should get project.estimated_cost
+ */
```

Comment:
I haven't thought about giving more detail about the props which receive the component but I like your consideration to give a better context for anyone who read it. Thanks

Reply:
;)
This are [JSDoc](https://devdocs.io/jsdoc/) comments. I added no libraries, just a procedure to add content and documentation to describe functions and requirements. There's a module to extract all the info an compile it as documentation.
Seems quite wasteful in time but,to think about the parameters  helps design a better function, like KISS, YAGNI,